client/lib/titan/new-mailbox.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * External dependencies
+ */
+import emailValidator from 'email-validator';
+import PropTypes from 'prop-types';
+import { translate } from 'i18n-calypso';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Internal dependencies
+ */
+import { validatePasswordField } from 'calypso/lib/gsuite/new-users';
+
+const valueIsBoolean = ( value ) => typeof value === 'boolean';
+const valueIsNonEmpty = ( value ) => value !== '';
+const valueIsOptional = () => true;
```

Comment:
Sorry for the dumb question (and a late one too), but I stumbled upon this and was wondering: is this always expected to return `true`? Shouldn't it return `true` only if the field is non-required?